test/lvm.sh

```diff
@@ -39,7 +39,10 @@ die() {
 }
 
 test_lvm() {
-    which vgcreate || echo "===> SKIPPING lvm backing: vgcreate not found" && return
+    if [ ! `which vgcreate` ]; then
```

Comment:
No need to run through test here.

    if ! which vgcreate >/dev/null; then

Will save you a fork()